packages/core/src/components/text-input/text-input.css

```diff
@@ -130,6 +130,7 @@
   color: var(--helper-color-invalid);
   display: block;
   font-size: var(--fontsize-body-m);
+  line-height: var(--lineheight-l);
```

Comment:
Should we add line-height for helper-text too?

Reply:
Good point. I'll add line-height for it too.

- [ ] Add line-height for helper-text too.